src/misc/crypt/crypt_inits.c

```diff
@@ -37,6 +37,54 @@ void init_GMP(void)
 }
 #endif
 
+int crypt_mp_init(const char* mpi)
```

Comment:
I am still not sure if we need this. I am not strongly against, just in doubts.

Reply:
No we absolutely don't need it, but I thought it's better than the other init functions that exist, so we just remove them ASAP (that's also why I marked them already as deprecated)

Reply:
damn, now I know why this change is in this PR, I changed the base branch :(

Reply:
let's focus on the other PR first, I've still to get this one to build